core/src/main/java/com/megaease/easeagent/core/Bootstrap.java

```diff
@@ -90,16 +88,10 @@ public static void start(String args, Instrumentation inst, String javaAgentJarP
             LOGGER.debug("Injected class: {}", bootstrapClassSet);
         }
 
-        // initiate configuration
-        String configPath = System.getProperty(AGENT_CONFIG_PATH);
-        if (StringUtils.isEmpty(configPath)) {
-            configPath = args;
-        }
-
         ClassLoader classLoader = Bootstrap.class.getClassLoader();
         final AgentInfo agentInfo = AgentInfoFactory.loadAgentInfo(classLoader);
         EaseAgent.agentInfo = agentInfo;
-        final GlobalConfigs conf = ConfigFactory.loadConfigs(configPath, classLoader);
+        final GlobalConfigs conf = ConfigFactory.loadConfigs(classLoader);
```

Comment:
The reason for passing a path from Bootstrap.java is to enhance code readability.  It allows us to control and clearly see in the entry point where the file being read is coming from.  This part cannot be changed.

Reply:
It is beneficial to centralize the configuration of agent-related settings into a single config component. This approach improves code readability and maintainability by avoiding scattered configurations throughout the codebase. This consolidation includes settings such as easeagent.config.path, easeagent.server.port, and easeagent.server.enabled. By using config.get(key) as the sole method to retrieve configuration values, it becomes the unified source for accessing configurations. This simplifies the management of configurations and promotes a more organized and maintainable codebase.

Reply:
> It is beneficial to centralize the configuration of agent-related settings into a single config component. This approach improves code readability and maintainability by avoiding scattered configurations throughout the codebase. This consolidation includes settings such as easeagent.config.path, easeagent.server.port, and easeagent.server.enabled. By using config.get(key) as the sole method to retrieve configuration values, it becomes the unified source for accessing configurations. This simplifies the management of configurations and promotes a more organized and maintainable codebase.

No, "config" itself comes from "easeagent.config.path". You put "easeagent.config.path" into "config" and get "easeagent.config.path" from "config". This It is an act of confusion.
We need concise code: get "config" from "easeagent.config.path", then get easeagent's configuration from "config"